api/v1/external/google/protobuf/timestamp.proto

```diff
@@ -1,138 +0,0 @@
-// Protocol Buffers - Google's data interchange format
```

Comment:
How does files generation work now that these external proto definitions have been removed?

Reply:
timestamp and wrappers are distributed with protobuf package and don't need to be vendored through custom `-I`